packages/brain/src/modules/apiServers/launchpad/validation/requestValidation.ts

```diff
@@ -0,0 +1,73 @@
+import { tags as availableTags, isValidBlsPubkey, isValidEcdsaPubkey, Tag, Web3signerDeleteRequest } from "@stakingbrain/common";
```

Comment:
In terms of organization for me does not make sense to have this code separated from where its being used. 

Instead, I would create two module insider routes: `feeRecipients` and `keystores`, then move these utility functions to where correspond.

Reply:
How would it look like?
```
routes/
   feeRecipients/
      index.ts
      route.ts
      validation.ts
   keystores/
      index.ts
      route.ts
      validation.ts
 ```
 
 Like this?

Reply:
yes